### PR TITLE
MF-776 and MF-715: Changes in the medications tab in the patient chart.

### PR DESCRIPTION
**Pull request by vasharma05**
### What does this PR do?
This PR pushes design changes to the orders tab.

Iterations: 
1. [MF-766: Drug Indication, Quantity, and Refills do not appear as required by wireframes](https://issues.openmrs.org/browse/MF-766) - Completed
2. [MF-715: Patient Instructions should show up in Medications Details](https://issues.openmrs.org/browse/MF-715) - Todo

Design:  https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d5e120913c6a092274f240

#### Iteration 1

Before: 
image

After:
image

### CODE REVIEW

File: packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx [hunk 1]
```diff
@@ -149,7 +149,7 @@ const MedicationsDetailsTable = connect<
                     {row.cells.map((cell) => (
                       <TableCell key={cell.id}>{cell.value?.content ?? cell.value}</TableCell>
                     ))}
-                    <TableCell className="bx--table-column-menu">
+                    <TableCell className="bx--table-column-menu" style={{ padding: '0.5rem 0' }}>
```
Comment (FlorianRappl) [inline packages/esm-patient-medications-app/src/components/medications-details-table.component.tsx:152]: Can we make this change via a class in SCSS?

File: packages/esm-patient-medications-app/src/utils/get-dosage.ts [hunk 2]
```diff
@@ -1,26 +1,24 @@
-export function getDosage(strength, doseNumber) {
+export function getDosage(strength: string, doseNumber: number) {
   if (!strength || !doseNumber) {
     return '';
   }
 
   const i = strength.search(/\D/);
-  const strengthQuantity = strength.substring(0, i);
+  const strengthQuantity = parseInt(strength.substring(0, i));
```
Comment (FlorianRappl) [inline packages/esm-patient-medications-app/src/utils/get-dosage.ts:7]: Either have it as `+...` or use `parseInt(..., 10)`

File: packages/esm-patient-medications-app/src/utils/get-dosage.ts [hunk 3]
```diff
@@ -1,26 +1,24 @@
...
   const concentrationStartIndex = strength.search(/\//);
 
   let strengthUnits = strength.substring(i);
-  let dosage;
 
   if (concentrationStartIndex >= 0) {
     strengthUnits = strength.substring(i, concentrationStartIndex);
     const j = strength.substring(concentrationStartIndex + 1).search(/\D/);
-    const concentrationQuantity = strength.substr(concentrationStartIndex + 1, j);
+    const concentrationQuantity = parseInt(strength.substr(concentrationStartIndex + 1, j));
```
Comment (FlorianRappl) [inline packages/esm-patient-medications-app/src/utils/get-dosage.ts:16]: Same here. I'd say let's prefer `+`.

**Review by FlorianRappl — APPROVED** [review]
LGTM!

**Comment by vasharma05** [comment]
@gracepotma the work to [MF-766](https://issues.openmrs.org/browse/MF-766) and [MF-715](https://issues.openmrs.org/browse/MF-715) is completed. Please have a look at the designs and the screenshots attached. 

Thanks!

**Review by FlorianRappl — APPROVED** [review]
LGTM - thanks! 🍻

**Pull request merged**